### DISCUSSION TITLE
influxdb v2

### DISCUSSION THREAD

**Discussion by lollul** [basking-in-the-sun2000/solar-logger#52] [Ideas]
Wondering if anyone tried upgrading influx to v2. I'm tempted to check out flux and its ability to generate accurate monthly grouping. I realize the continuous query would need to be converted. Should I expect any other obstacles?

**Comment by basking-in-the-sun2000** [comment]
it is on the top of changes, however, it is a major one. v1.8 doesn't have a good flux and would need to change all the calls to flux before migrating to v2. However, to test the calls I need v2, so a bit of a catch-22
